tests/many_to_one/models.py

```diff
@@ -116,3 +116,18 @@ class School(models.Model):
 
 class Student(models.Model):
     school = models.ForeignKey(School, models.CASCADE)
+
+
+# Joins trimming (#26551)
```

Comment:
If possible to reuse at least some models, that's ideal. Ticket references aren't really needed since they might be used for other tickets later on.